ofronds_internals/test/dune

```diff
@@ -0,0 +1,5 @@
+(cram
+ (deps
+  %{bin:ofronds}
+  (package ofronds)
+  (source_tree .)))
```

Comment:
Again, not necessary at the moment since all the cram tests all generate their own files from scratch. I've found that longer-term it can work better to have the files already existing, in which case I'd add this to ensure that Dune copies them over.